Frameworks/Foundation/NSFileManager.mm

```diff
@@ -302,18 +302,18 @@ - (BOOL)createDirectoryAtPath:(NSString*)pathAddr
                 bool success = EbrMkdir(curPath);
                 if (!success && errno != EEXIST) {
                     TraceError(TAG, L"Failed to make path %hs: %d", curPath, errno);
-                    // return FALSE;
+                    // return NO;
```

Comment:
Is there a reason this is commented out and not just removed? Also, it seems like the error parameter is unused. This is probably outside the scope of this review, but could you take a look at it and file issues if necessary or follow up with somebody?